views/openacademy_menu.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- ACTIONS OPENING VIEWS ON MODELS -->
+    <record id="action_window_course" model="ir.actions.act_window">
+      <field name="name">openacademy.menu_course</field>
+      <field name="res_model">openacademy.course</field>
+      <field name="view_mode">tree,form</field>
```

Comment:
Action says tree, but it seems you didn't defined any.

Same for the below action.